docs/git-resolver.md

```diff
@@ -145,6 +145,10 @@ spec:
   * Gitea
   * BitBucket Server
   * BitBucket Cloud
+- As users can specify branch name or tag name for the param `revision `, the actual `commit sha` being used at the moment of resolving the resource is recorded in the `status.annotations` of both `ResolutionRequest` and `PipelineRun`/`TaskRun` object along with the `url` and `path`. The annotations keys are 
```

Comment:
Can we put this under its own heading with an example?

Reply:
Sure! Added PTAL1

Reply:
Hey @dibyom ,

I breakdown this PR for easier review, and the change for the git resolver is now in https://github.com/tektoncd/pipeline/pull/5664. I've implemented this feedback in that PR. PTAL. Thank you!!